core/models/building.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Building(models.Model):
```

Comment:
Let's refer to the model as `OfficeBlock` rather than `Building`. Pardon that this was not stated clearly on the acceptance criteria. Should be a minor refactor.

Reply:
Also, while at it...rename the model field from `block_name` to `name`. See number *15* [here](https://goo.gl/6gqMkb) to understand why.